gui/public/i18n/en/translation.ftl

```diff
@@ -135,7 +135,34 @@ body_assignment_menu-manage_trackers = Manage all trackers
 body_assignment_menu-unassign_tracker = Unassign tracker
 
 ## Tracker assignment menu
-tracker_selection_menu-title = Which tracker to assign to the { $bodyPart }?
+
+# A -translation_key (with a dash in the front) means that it's a label.
+# It can only be used in the translation file, it's nice for reusing names and that kind of stuff.
+# 
+# We are using it here because english doesn't require changing the text in each case but
+# maybe your language does.
+-tracker_selection-part = Which tracker to assign to your
+tracker_selection_menu-title-NONE = { -tracker_selection-part } unassigned
```

Comment:
uhhh: `Which tracker to assign to your unassigned`
wat

Reply:
What I want to know is if its even possible to get this query

Reply:
because there is no logical way to put this lol

Reply:
Hope you like this one :P